backend/notification/socket.js

```diff
@@ -54,35 +33,6 @@ const createSocketServer = (app) => {
         }
       }
     });
-    
-    socket.on(socket_names.added_user_to_group, async (data) => {
-      const { userId, message } = data;
-
-      try {
-        await rateLimiter.useRateLimiter(userId);
-        const userSocketId = userSocketMap.get(userId);
-        if (userSocketId) {
-          userSocketId.emit(socket_names.added_user_to_group, { message });
-        }
-      } catch (rate_limit_message) {
-        socket.emit(socket_names.rate_limit, rate_limit_message);
-      }
-    });
-
-    socket.on(socket_names.group_notifications, async (data) => {
-      const { userIds, message } = data;
-      userIds.forEach((userId) => {
-        const userSocketId = userSocketMap.get(userId);
-        if (userSocketId) {
-          try {
-            rateLimiter.useRateLimiter(userId);
-            userSocketId.emit(socket_names.group_notifications, { message });
-          } catch (rate_limit_message) {
-            socket.emit(socket_names.rate_limit, rate_limit_message);
-          }
-        }
-      });
-    });
```

Comment:
Why isn't this needed anymore?

Reply:
I read that since I am directly emitting notifications from the controller files, I do not need to specify this in the main socket.js file. I tested it and it works the same. The video demonstration was done without it.